test/com/google/enterprise/adaptor/prebuilt/PrebuiltTransformsTest.java

```diff
@@ -69,13 +70,13 @@ public void testCopyMetadataOverwriteFalse() {
     metadata.add("author", "Fred");
     metadata.add("contributors", "Mary");
     metadata.add("contributors", "George");
-    transform.transform(metadata, new HashMap<String, String>());
+    transform.transform(metadata, new LinkedHashMap<String, String>());
```

Comment:
In general, we perfer to minimize diffs, and the metadata and params maps, and their expected results (which are asserted with Map.equals), don't need to be ordered. In fact it is probably better if they aren't.  It is probably safe to have have done:

s/config = new HashMap/config = new LinkedHashMap/g